languages/ruby/spec/oso/polar/data_filtering/new_filter_spec.rb

```diff
@@ -35,12 +32,12 @@ class Planet < ActiveRecord::Base
             Person,
             fields: {
               name: String,
-              sign_name: String,
+              sign_id: Integer,
```

Comment:
These were changed so there would be a numeric field for testing inequality operators